packages/react/.storybook/static/assets/images/avatar-menu.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
lets rename this to `avatar-(${gender} | ${give a cool name}).svg`.

Reply:
Fixed in latest commits